capabilities.json

```diff
@@ -307,6 +307,10 @@
             ]
           }
         },
+        "show_day": {
+          "displayName" : "Show Day",
+          "type" : { "bool" : true }
+        },
```

Comment:
Instead of an additional boolean option, can you change this to an option in the `"date_format_day"` enumeration:

```
{ "displayName" : "(None)", "value" : "none" }
```

Reply:
Done - I'm passing a different value instead of a string "none" to hopefully be more descriptive.